js/main.js

```diff
@@ -447,3 +446,14 @@ function setupShareCopy(){
         Materialize.toast('Failed to copy share link!', 1000);
     });
 }
+
+function clearAll(){
+    console.log("remove all");
+
+    player.stopVideo();
+    var id;
+    for(var i=0; i<playlist.length; i++){
+        id=playlist[i].id.videoId;
+        $("#playlist #"+id).remove();
+    }
```

Comment:
I think `playlist = []` is required here.

Reply:
Also, `.remove()` doesn't animate consistently with what `removeFromPlaylist()` does. You can take the animation part out into a separate function, and call that for each element instead of `remove()`.

Reply:
But if you are removing everything, it happens more or less instantly. Do you think an animation step is necessary?